gitpod-io/gitpod-vscode-desktop
# Bring telemetry inside proxy IDE-142

## Description

- Add common telemetry https://github.com/gitpod-io/gitpod-vscode-desktop/pull/70#discussion_r1217735382
- Bring telemetry inside proxy itself

## Related Issue(s)

Fixes #

## How to test


✅  See test results here https://github.com/gitpod-io/gitpod-vscode-desktop/pull/72#issuecomment-1576538418

- Add code below to `extension.ts` and `proxy.ts`, and change logger in proxy.ts
```
telemetryService.sendTelemetryEvent('hello>>>>>>>', { gitpodHost: 'https://gitpod.io', a: '11111'});
telemetryService.sendTelemetryException(new Error('test aa /Users/hwen/aaa/aaa.log'), { gitpodHost: 'https://gitpod.io', a: '22222' });
```
- Run `yarn package` and install `*.visx` file locally
- Reload extension, check its output
- Open Gitpod with VSCode Destkop, check `/tmp/lssh.log` file

## Documentation


/hold
